src/sentry/conf/server.py

```diff
@@ -1405,6 +1405,14 @@ def get_sentry_sdk_config():
 
 SENTRY_SDK_CONFIG = get_sentry_sdk_config()
 
+# Callable to bind additional context for the Sentry SDK
+#
+# def get_org_context(scope, organization, **kwargs):
+#    scope.set_tag('organization.cool', '1')
+#
+# SENTRY_ORGANIZATION_CONTEXT_HELPER = get_org_context
+SENTRY_ORGANIZATION_CONTEXT_HELPER = None
```

Comment:
Are you planning to use this down the road or in getsentry?

Reply:
Yeah it’s the primary reason for the PR (see the comments)